onnx_caffe2/backend.py

```diff
@@ -715,21 +776,26 @@ def _onnx_model_to_caffe2_net(cls, onnx_model, device, opset_version, include_in
         predict_model.ParseFromString(cls.optimize_onnx(onnx_model.SerializeToString(), predict=True))
         cls._inplace_rewrite(predict_model.graph)
 
-        init_net = caffe2_pb2.NetDef()
-        predict_net = caffe2_pb2.NetDef()
+        # We use two separate ModelHelpers, so that the rest of the
+        # code doesn't have to worry about whether to add to
+        # param_init_net or net.
+        init_model_helper = ModelHelper(name=onnx_model.graph.name+'_init')
```

Comment:
remove _init?

Reply:
why? This matches the behavior before the diff (see the red code on the left)

Reply:
because the model helper will name the net 'graph.name' and 'graph.name+"_init"': https://github.com/caffe2/caffe2/blob/master/caffe2/python/model_helper.py#L118

Reply:
++

resolved by not using ModelHelper anymore